reference/robot.md

```diff
@@ -356,9 +359,12 @@ function `getDistanceSensor` will return a reference to a
 found, the function returns `NULL` in C++, `null` in Java or the `none` in
 Python.
 
+> Note: These functions are not available in the C and MATLAB APIs. Instead, C and Matlab users should use [wb\_robot\_get\_device](#wb_robot_get_device).
```

Comment:
The rendering of this link is wrong: see https://www.cyberbotics.com/doc/reference/robot?version=fix-robot-doc#wb_robot_get_device

robot appears in *italics* and some underscores are missing...